src/Network/Probe.php

```diff
@@ -47,6 +47,31 @@ class Probe
 	private static $baseurl;
 	private static $istimeout;
 
+	/**
+	 * Remove stuff from an URI that doesn't belong there
+	 *
+	 * @param string $URI
+	 * @return string Cleaned URI
+	 */
+	public static function cleanURI(string $URI)
```

Comment:
Please rename to `cleanProfileUri` or `cleanProfileURI` since it's more specific than just general URIs.

Additionally, variables should be `camelCased`, which would mean `$uri` even if it represents an acronym.